eng/sdl-tsa-vars.config

```diff
@@ -3,7 +3,7 @@
 -TsaProjectName DEVDIV
 -TsaNotificationEmail aspnetcore-build@microsoft.com
 -TsaCodebaseAdmin REDMOND\kevinpi
--TsaBugAreaPath "DevDiv\ASP.NET Core"
```

Comment:
@dougbu, @wtgodbe I've noticed that there is a difference between `\` and `\\` in these two files. I'd like to just use the same `\\` in both files. Do you know if that may cause any issues?

Reply:
@dotnet/dnceng would know for sure, but I suspect `\\` here will be literal, which may cause problems with area assignment

Reply:
I'm not sure what is using the tsaoptions.json, but the two files are read by two different processes which probably interpret string literals differently. The sdl-tsa-vars.config file is read by the release pipeline as a text file and doesn't need the slashes to be escaped, where the tsaoptions.json file probably does require escaping.

Reply:
Yes, I believe that's correct. Having two slashes here will produce two slashes in the TSA configuration file, and I'm not sure that TSA will handle it properly.

Reply:
Good that I've checked then 👍 
I'll leave this as is.